README.md

```diff
@@ -138,6 +138,23 @@ By default, fields are assumed to be of type ```:string```. Other built-in types
 If built-in types do not suit you, you can use a custom field type represented by an arbitrary class, provided that the class supports a compatible serialization interface.
 The primary use case for using a custom field type is to represent your business logic with high-level types, while ensuring portability or backward-compatibility of the serialized representation.
 
+#### Note on boolean type
+
+The boolean fields are stored as `"t", "f"` strings by default. DynamoDB
+supports boolean type natively. So if you want to use native boolean
+type or already have table with native boolean attribute you can easily
+acheive this with `store_as_boolean` option:
```

Comment:
Typo `acheive` -> `acheive`.

Also could we change the naming to `store_as_native_boolean: true` or shorter `native_boolean: true` That might call more to action that it's the underlying DynamoDB layer of storage?

Reply:
That was my typo :( . Thanks for fixing!